torch/serialization.py

```diff
@@ -656,6 +656,10 @@ def persistent_id(obj):
     data_value = data_buf.getvalue()
     zip_file.write_record('data.pkl', data_value, len(data_value))
 
+    # Write byte encoding marker
+    byteencodingvalue = b'le' if sys.byteorder == 'little' else b'be'
+    zip_file.write_record('byteencoding', byteencodingvalue, len(byteencodingvalue))
```

Comment:
I'd suggest calling the record `'byteorder'`.

Would it be simpler to just set `byteencodingvalue = sys.byteorder`? It's only a few extra characters.

Either way, the name of the record and its possible values will be somewhat set in stone after this PR is merged, since changing them in the future would have compatibility issues

Reply:
Sure, I'll rename record.

As for using sys.byteorder, if it is guaranteed to not change to something else in newer versions of python, then it's definitely fine. I'll update it as well.

Reply:
It might not necessarily be guaranteed to stay the same. But if it ever is changed or other possible values of `sys.byteorder` are added in the future for whatever reason, it will error out, people will report it to us or contributors will find out about, and we can fix it when that happens. That's probably the best we can do, and I think it's better than silently defaulting to something and potentially being incorrect